webclient/components/pt-info/single/1time-1row-mField/phq9/db/client-side/structure/patient-table-of-phq9.js

```diff
@@ -0,0 +1,39 @@
+import clientSideTableManage from '~/components/core/crud/manage-rows-of-table-in-client-side-orm.js'
+import phq9MasterClass from './master-table-of-phq9.js'
+
+const { v1: uuidv1 } = require('uuid')
+
+let count = 0
+const intUniqueID = () => ++count
+
+export default class phq9ForPatientClass extends clientSideTableManage {
+  static entity = 'tblPhq9Patient'
+  static apiUrl = 'http://localhost:8000/public/api/phq9/v20/'
+  static primaryKey = 'clientSideUniqRowId'
+  static graphSeries1Unit = 'unit'
+
+  static fields() {
+    return {
+      ...super.fields(),
+
+      clientSideUniqRowId: this.uid(() => intUniqueID()),
+      patientUUID: this.uid(() => uuidv1()),
```

Comment:
1. Typo error - 'clientSideUniqRowId' should be 'clientSideUniqueRowId'
2. Value of 'patientUUID' should be this.string(null)
3. Add one more column 'serverSideRowUuid'